keps/sig-scheduling/902-non-preempting-priorityclass/README.md

```diff
@@ -199,18 +205,117 @@ Ensure existing tests (for preempting PriorityClasses) do not break.
 
 ### Rollout, Upgrade and Rollback Planning
 * **How can a rollout fail? Can it impact already running workloads?**
-  The scheduler errors and exits during start up. Existing workloads are not
-  affected.
+  If a rollout fails, kube-scheduler will keep crashing. Running workloads won't be affected by kube-scheduler.
 
 * **What specific metrics should inform a rollback?**
-  N/A.
+Check the following indicators to determine if there are any exceptions:
+  - pod_preemption_victims
+  - total_preemption_attempts
+  - scheduling_algorithm_preemption_evaluation_seconds
 
 * **Were upgrade and rollback tested? Was upgrade->downgrade->upgrade path tested?**
-  N/A.
+  Manually tested successfully.
```

Comment:
Here you should describe the test and environment were you performed this.

Reply:
Done